### PR TITLE
Make lazyload working again

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [x] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue resolved: N/A

Images inside `.content` element already have `src` attribute. Browsers will load them by default, so `lazyload: true` takes no effect.

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by jiangtj** [comment]
How about use [lozad](https://github.com/ApoorvSaxena/lozad.js) or [lazysizes](https://github.com/aFarkas/lazysizes) instead of it ?

**Comment by stevenjoezhang** [comment]
Good idea. Currently, lazyload relies on jQuery and has some issues - one image makes two network requests

**Comment by stevenjoezhang** [comment]
I didn't use these two lazyload plugins before. What are the differences in functionality and performance between them? Seems lazysizes has more stars / forks.

**Comment by jiangtj** [comment]
lazysizes is a full-featured lazy loading library that lazy loads images and iframes. The pattern it uses is quite similar to the code examples shown here in that it automatically binds to a lazyload class on  elements, and requires you to specify image URLs in data-src and/or data-srcset attributes, the contents of which are swapped into src and/or srcset attributes, respectively. It uses intersection observer (which you can polyfill), and can be extended with a number of plugins to do things like lazy load video.

lozad.js is a super lightweight option that uses intersection observer only. As such, it's highly performant, but will need to be polyfilled before you can use it on older browsers.

--- from [google web developers](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/)

**Comment by stevenjoezhang** [comment]
I see. I also support using the lightweight one - lozad.js

**Comment by jiangtj** [comment]
Need add a options to load polyfill for ie user. 
https://github.com/ApoorvSaxena/lozad.js#browser-support

File: source/js/utils.js [hunk 1]
```diff
@@ -46,11 +46,7 @@ NexT.utils = NexT.$u = {
   },
 
   lazyLoadPostsImages: function() {
-    $('#posts').find('img').lazyload({
-      //placeholder: '/images/loading.gif',
-      effect   : 'fadeIn',
-      threshold: 0
-    });
+    lozad('.content img').observe();
```
[the remark this answers is not in the document]
Comment (jiangtj) [reply source/js/utils.js]: image
try this

[the remark this answers is not in the document]
Comment (stevenjoezhang) [reply source/js/utils.js]: Ignore this warning. `lozad` is `window.lozad`

[the remark this answers is not in the document]
Comment (jiangtj) [reply source/js/utils.js]: 😂 Okay

**Comment by stevenjoezhang** [comment]
No need to add an option to load polyfill for ie user. NexT v7 already dropped ie support.

File: _config.yml [hunk 2]
```diff
@@ -1041,11 +1041,11 @@ vendors:
   # fastclick: //cdnjs.cloudflare.com/ajax/libs/fastclick/1.0.6/fastclick.min.js
   fastclick:
 
-  # Internal version: 1.9.7
-  # See: https://github.com/tuupola/jquery_lazyload
+  # Internal version: 1.10.0
```
Comment (jiangtj) [inline _config.yml]: version: 1.9.0

Comment (stevenjoezhang) [reply _config.yml]: ? It is 1.10.0

Comment (jiangtj) [reply _config.yml]: image
😂 ???

Comment (stevenjoezhang) [reply _config.yml]: I created https://github.com/theme-next/theme-next-lazyload, and I'm quite sure I am using 1.10.0

Comment (stevenjoezhang) [reply _config.yml]: jsDelivr is also using this version..

Comment (stevenjoezhang) [reply _config.yml]: The internal version is 1.10.0, I just copy and paste files from 1.10.0 folder

Comment (jiangtj) [reply _config.yml]: image

Comment (stevenjoezhang) [reply _config.yml]: So what... you will get a 404 not found if you use 1.10.0 here.

File: scripts/tags/exturl.js [hunk 3]
```diff
@@ -1,62 +0,0 @@
...
-/* eslint-enable */
-
-function extURL(args) {
```
Comment (jiangtj) [inline scripts/tags/exturl.js]: Remove ok?

Comment (stevenjoezhang) [reply scripts/tags/exturl.js]: The warnings has been displayed for about 2 months. Removing it should be fine

**Comment by stevenjoezhang** [comment]
See also https://github.com/hexojs/hexo-renderer-marked/pull/156

**Pull request merged**